src/FsAutoComplete.Core/FsAutoComplete.Core.fsproj

```diff
@@ -1,20 +1,18 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
-
+<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFrameworks>net461;netstandard2.0</TargetFrameworks>
   </PropertyGroup>
-
   <PropertyGroup Condition=" '$(TargetFramework)' == 'net461' ">
-    <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
     <Prefer32Bit>false</Prefer32Bit>
-    <Platform>x64</Platform>
   </PropertyGroup>
-
   <PropertyGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' ">
     <DefineConstants>NO_EXTENSIONTYPING;NO_PROJECTCRACKER;SCRIPT_REFS_FROM_MSBUILD;SUAVE_2;$(DefineConstants);DOTNET_SPAWN</DefineConstants>
   </PropertyGroup>
-
   <ItemGroup>
+    <ProjectReference Include="..\FsAutoComplete.Core.VerboseSdkHelper\FsAutoComplete.Core.VerboseSdkHelper.fsproj" />
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Include="DotnetNewTemplate.fs" />
```

Comment:
chan you revert the changes to fhis file?
i expect to see only a new `<Compile Include="DotnetNewTemplate.fs" />`

Reply:
same issue after rebase